<launch>  
    <node pkg="bluefox2" type="bluefox2" name="bluefox2" output="screen" >
        <param name="fps" type="double" value="30.0" />
        <param name="gain" type="double" value="5.0" />
        <param name="use_auto_exposure" type="bool" value="false" />
        <param name="cam_cnt" type="int" value="3" />
        <param name="seriala" type="string" value="25000704" />   
        <param name="exposure_time_usa" type="int" value="10000" />     
        <param name="serialb" type="string" value="25000709" />   
        <param name="exposure_time_usb" type="int" value="10000" />     
        <param name="serialc" type="string" value="25001161" />   
        <param name="exposure_time_usc" type="int" value="10000" />     
        <param name="pub_cnt" type="int" value="1" />   
        <param name="mask0" type="string" value="abc" />   
    </node> 
</launch>  
